sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuCast.scala

```diff
@@ -1482,88 +1258,4 @@ object GpuCast {
       _.castTo(GpuColumnVector.getNonNestedRapidsType(toType))
     }
   }
-}
-
-/**
- * Casts using the GPU
- */
-case class GpuCast(
```

Comment:
Why move this class? It make checking for changes to the code a lot more difficult to do.